ui/package.json

```diff
@@ -20,7 +20,7 @@
     "@lingui/react": "^4.13.0",
     "@mui/icons-material": "^5.16.7",
     "@mui/lab": "5.0.0-alpha.132",
-    "@mui/material": "^5.13.3",
+    "@mui/material": "5.14.11",
```

Comment:
What's the reason for this change?

Reply:
That's the highest version we can use before the breaking change, so I pinned it at that.
https://github.com/wmcso/artemis/wiki/Updating-Dependencies/#held-back-updates